### PR TITLE
Finish rebranding

### PR DESCRIPTION
**Pull request by mikebattista**
# PR checklist

**Quick summary of changes**
See individual checkins for descriptions of changes. Mostly search and replace operations.

I'm not finished auditing everything, but since this touches lots of files, I'm trying to break it up into manageable chunks that can be reviewed, validated, and merged safely. Exhaustive test passes will be run as part of signing off on the 1.0 release. I created a few projects with the different extensions and ran the CI pipelines which all passed.

**Which issue does this PR relate to?**
#4396

**Applies to the following platforms:**

- [x] WinUI
- [x] WPF
- [x] UWP

**Anything that requires particular review or attention?**

**Do all automated tests pass?**

**Have automated tests been added for new features?**

**If you've changed the UI:**
  - Be sure you are including screenshots to show the changes.
  - Be sure you have reviewed the [accessibility checklist](accessibility.md).

**If you've included a new template:**
  - Be sure you reviewed the [Template Verification Checklist](https://github.com/microsoft/WindowsTemplateStudio/wiki/Checklist:-Template-Verification).
  - Be sure it's included in the list on this [UWP](https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/UWP/getting-started-endusers.md) or [WPF](https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/WPF/getting-started-endusers.md) getting started doc.

**Have you raised issues for any needed follow-on work?**

**Have docs been updated?**

**If breaking changes or different ways of doing things have been introduced, have they been communicated widely?**

### CODE REVIEW

File: _build/Extension-SetIdentityAndVersion.ps1 [hunk 1]
```diff
@@ -29,7 +29,7 @@ Param(
   [string]$cmdSetGuid,
 
   [Parameter(Mandatory=$False,Position=10)]
-  [string]$targetVsixCommandMenuName = "Windows Template Studio (local)",
```
Comment (mrlacey) [inline _build/Extension-SetIdentityAndVersion.ps1:32]: Not sure this file is still needed. If it is we don't use the "(local)" suffix for anything anymore.

File: code/SharedFunctionality.UI/Services/ProjectMetadataService.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public static class ProjectMetadataService
         private const string VersionAttribLiteral = "Version";
         private const string ItemLiteral = "Item";
 
-        private static readonly XNamespace NS = "http://schemas.microsoft.com/appx/developer/windowsTemplateStudio";
+        private static readonly XNamespace NS = "http://schemas.microsoft.com/appx/developer/templatestudio";
```
Comment (mrlacey) [inline code/SharedFunctionality.UI/Services/ProjectMetadataService.cs:31]: This change is likely to cause issues if trying to add something to an app that was created in the VS2019 version of the extension.
If that scenario is in scope then may need to reconsider this change.

File: code/SharedFunctionality.UI/Services/ProjectMetadataService.cs [hunk 3]
```diff
@@ -38,12 +38,12 @@ public static ProjectMetadata GetProjectMetadata(string projectPath)
...
                 if (!string.IsNullOrEmpty(projectPath))
                 {
-                    var metadataFileNames = new List<string>() { "Package.appxmanifest", "WTS.ProjectConfig.xml" };
+                    var metadataFileNames = new List<string>() { "Package.appxmanifest", "TemplateStudio.xml" };
                     var metadataFile = metadataFileNames.FirstOrDefault(fileName => File.Exists(Path.Combine(projectPath, fileName)));
                     if (!string.IsNullOrEmpty(metadataFile))
                     {
                         var manifest = XElement.Load(Path.Combine(projectPath, metadataFile));
-                        XNamespace ns = "http://schemas.microsoft.com/appx/developer/windowsTemplateStudio";
+                        XNamespace ns = "http://schemas.microsoft.com/appx/developer/templatestudio";
```
Comment (mrlacey) [inline code/SharedFunctionality.UI/Services/ProjectMetadataService.cs:46]: It seems odd that this line is even in the file as it's a duplicate from line 31 (ignoring case)

File: code/SharedFunctionality.UI/Services/ProjectMetadataService.cs (hunk 2 above)
Comment (mikebattista) [reply code/SharedFunctionality.UI/Services/ProjectMetadataService.cs:31]: That scenario is not in scope. This is a generational leap to VS 2022 and we should take the opportunity to move the project and branding forward. Want to do this now so we don't break update scenarios in the future for projects built with the new extensions.

**Review by mrlacey — APPROVED** [review]
Assuming that almost all changes were done with a global search & replace so won't bother checking everything manually

File: code/SharedFunctionality.UI/Services/ProjectMetadataService.cs (hunk 3 above)
Comment (mikebattista) [reply code/SharedFunctionality.UI/Services/ProjectMetadataService.cs:46]: Going to merge as is for now.

**Pull request merged**